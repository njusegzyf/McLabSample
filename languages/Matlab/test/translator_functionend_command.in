function a
cmd end function end
end