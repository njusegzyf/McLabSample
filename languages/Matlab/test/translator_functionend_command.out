>>>> Destination Language -> Source Language
[1, 1] -> (1, 1)
[1, 2] -> (1, 2)
[1, 3] -> (1, 3)
[1, 4] -> (1, 4)
[1, 5] -> (1, 5)
[1, 6] -> (1, 6)
[1, 7] -> (1, 7)
[1, 8] -> (1, 8)
[1, 9] -> (1, 9)
[1, 10] -> (1, 10)
[1, 11] -> (1, 11)
[2, 1] -> (2, 1)
[2, 2] -> (2, 2)
[2, 3] -> (2, 3)
[2, 4] -> (2, 5)
[2, 5] -> (2, 5)
[2, 6] -> (2, 5)
[2, 7] -> (2, 6)
[2, 8] -> (2, 7)
[2, 9] -> (2, 7)
[2, 10] -> (2, 7)
[2, 11] -> (2, 9)
[2, 12] -> (2, 9)
[2, 13] -> (2, 10)
[2, 14] -> (2, 11)
[2, 15] -> (2, 12)
[2, 16] -> (2, 13)
[2, 17] -> (2, 14)
[2, 18] -> (2, 15)
[2, 19] -> (2, 16)
[2, 20] -> (2, 16)
[2, 21] -> (2, 16)
[2, 22] -> (2, 18)
[2, 23] -> (2, 18)
[2, 24] -> (2, 19)
[2, 25] -> (2, 20)
[2, 26] -> (2, 20)
[2, 27] -> (2, 20)
[2, 28] -> (2, 21)
[3, 1] -> (3, 1)
[3, 2] -> (3, 2)
[3, 3] -> (3, 3)
>>>> Translated Text
function a
cmd('end','function','end')
end